### PR TITLE
[RTL] About page doesn't support the RTL direction exactly. 

### PR DESCRIPTION
**Pull request by mermi**
This is ready, please review, I filled a Bug here #1180558

### CODE REVIEW

File: media/css/mozorg/about-base.less [hunk 1]
```diff
@@ -131,6 +131,10 @@ html[dir="rtl"] {
         margin-right: 0;
         margin-left: 490px;
     }
+    #main-content .intro {
+        float: right;
+
```
Comment (flodolo) [inline media/css/mozorg/about-base.less]: Style consistency: no empty line in the middle of the rule, empty line before the rule.

Comment (schalkneethling) [inline media/css/mozorg/about-base.less]: I believe the empty line should be `clear: right;` @mermi ?

Comment (schalkneethling) [inline media/css/mozorg/about-base.less]: although it does not seem to be needed

**Comment by schalkneethling** [comment]
Other than removing the one empty line, this is an r+ Thanks @mermi \o/

**Comment by schalkneethling** [comment]
Also, just FYI, naming your branches something like bug-bugnumber-bug-title and then making your commit message something like 'Fix Bug bugnumber, some descriptive message' enables the bot to comment in the bug and auto resolved fix it for you.

So this one could be:

bug-1180558-about-page-rtl, and the commit message 'Fix Bug 1180558, aligning video correctly fixes RTL'

**Comment by mermi** [comment]
ok, I will rename the branch @schalkneethling and @flodolo I am checking the empty line

File: media/css/mozorg/about-base.less (hunk 1 above)
Comment (mermi) [inline media/css/mozorg/about-base.less]: Yes it's for clear: right @schalkneethling

**Pull request closed without merging**